recipes/mimalloc/all/conanfile.py

```diff
@@ -111,13 +112,11 @@ def validate(self):
 
         if self.info.settings.compiler.cppstd:
             check_min_cppstd(self, self._min_cppstd)
-        check_min_vs(self, 191)
-        if not is_msvc(self):
-            minimum_version = self._compilers_minimum_version.get(str(self.info.settings.compiler), False)
-            if minimum_version and Version(self.info.settings.compiler.version) < minimum_version:
-                raise ConanInvalidConfiguration(
-                    f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
-                )
+        minimum_version = self._compilers_minimum_version.get(str(self.info.settings.compiler), False)
```

Comment:
This pr needs some attention it's been so long

```suggestion
        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
```